_config.yml

```diff
@@ -858,16 +858,16 @@ vendors:
   # Available values: local, jsdelivr, unpkg
   internal: local
 
-  # Internal version: 3.1.0
```

Comment:
`Internal version` means the version in `source/lib`: https://github.com/next-theme/hexo-theme-next/tree/master/source/lib
These files are not changed so it's not necessary to update the version.